Description: historyform: Fix sorting by time of event
 This patch backports commit a072b933032647cd2feca45adb4c11fa80410af6.
Author: Peter Colberg <peter@colberg.org>
Forwarded: not-needed
Last-Update: 2015-12-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/gui/historyform.cpp
+++ b/src/gui/historyform.cpp
@@ -27,6 +27,7 @@
 #include "qicon.h"
 #include "audits/memman.h"
 #include "historyform.h"
+#include <QDateTime>
 
 #define HISTCOL_TIMESTAMP 	0
 #define HISTCOL_DIRECTION	1
@@ -174,8 +175,7 @@ void HistoryForm::loadHistory()
             {
                 case HISTCOL_TIMESTAMP:
                 {
-                    QString time = QString::fromStdString(time2str(cr->time_start,  "%d %b %Y %H:%M:%S"));
-                    m_model->setData(index, time);
+                    m_model->setData(index, QDateTime::fromTime_t(cr->time_start));
                     break;
                 }
                 case HISTCOL_DIRECTION:
